### DISCUSSION TITLE
HicRenderer config

### DISCUSSION THREAD

**Discussion by yumisims** [GMOD/jbrowse-components#2963] [General] [answered]
Hi, 
Anyone has an example of HicRenderer config, I would like to know how does it work using jbrowse cli. Thank you

**Comment by cmdcolin** [comment]
Is there anything in particular about the HicRenderer that you want to customize? the HicRenderer config options are baseColor (which by default is red) and color (which adds alpha transparency to the baseColor depending on the score)

**Comment by cmdcolin** [comment, marked as answer]
here is an example of a hic track with a blue color scheme

```
{
  type: 'HicTrack',
  trackId: 'hic-1652980837207-sessionTrack',
  name: 'Hic Track (copy)',
  assemblyNames: ['hg19'],
  adapter: {
    type: 'HicAdapter',
    hicLocation: {
      locationType: 'UriLocation',
      uri: 'https://s3.amazonaws.com/igv.broadinstitute.org/data/hic/intra_nofrag_30.hic',
      baseUri: 'http://localhost:3000/test_data/config_demo.json',
    },
  },
  displays: [
    {
      type: 'LinearHicDisplay',
      displayId: 'hic-LinearHicDisplay-1652980837207',
      renderer: { type: 'HicRenderer', baseColor: 'rgba(16,0,255,1)' },
    },
  ],
}
```

I got this by making a copy of an existing track in the UI, editing the config, and then copying the config as JSON from the developer console using `JSON.stringify(getSnapshot(JBrowseSession.sessionTracks[0]))`

**Reply by yumisims** [reply]
thanks Colin, I got it working now, also good tip to use the developer console.
